app/src/main/java/org/fossasia/openevent/app/contract/presenter/LoginPresenter.java

```diff
@@ -0,0 +1,15 @@
+package org.fossasia.openevent.app.contract.presenter;
+
+import org.fossasia.openevent.app.contract.view.LoginView;
+
+public interface LoginPresenter {
+
+    void start();
```

Comment:
Analogous to attaching or detaching view lifecycle but more general for any starting activity and stopping activity to do cleanup and release view to avoid memory leaks